spark/src/main/scala/io/substrait/spark/expression/ToSubstraitExpression.scala

```diff
@@ -144,6 +144,7 @@ abstract class ToSubstraitExpression extends HasOutputStack[Seq[Attribute]] {
             "org.apache.spark.sql.catalyst.expressions.PromotePrecision") =>
         translateUp(p.children.head)
       case CaseWhen(branches, elseValue) => translateCaseWhen(branches, elseValue)
+      case InSet(value, set) => translateIn(value, set.toSeq.map(v => Literal(v)))
```

Comment:
nit: mind moving this next to the `case In(..)`?

Reply:
Sure, although `InSet` needs to come before `ScalarFunction` otherwise it matches the latter (since it's a `UnaryExpression`).  I've moved `In` up the list so they are together.